lib/nine_one_one.rb

```diff
@@ -27,8 +27,12 @@ def self.configs
     @configs ||= {}
   end
 
-  def self.emergency(incident_key, description, details_hash = nil)
-    use(:default).emergency(incident_key, description, details_hash)
+  def self.high_urgency_pager(description, source, details_hash = {})
+    use(:default).pager(description, source, details_hash, NineOneOne::PagerDutyService::HIGH_URGENCY_ERROR)
+  end
+
+  def self.low_urgency_pager(description, source, details_hash)
```

Comment:
I think that this change is coupling the interface of this gem with the technical details of PagerDuty API. I'd be in favour of keeping one, generic method and providing urgency as a optional argument which defaults to HIGH, rather that creating two methods. This way we would keep the genericity and not limit the abilities.